jax/_src/numpy/lax_numpy.py

```diff
@@ -3761,12 +3761,16 @@ def arange(start: core.DimSize, stop: Optional[core.DimSize]=None,
   lax._check_user_dtype_supported(dtype, "arange")
   require = partial(core.concrete_or_error, None)
   msg = "It arose in jax.numpy.arange argument `{}`.".format
+  if _any([core.is_special_dim_size(d) for d in (start, stop, step)]):
```

Comment:
nit, remove the square brackets and call `_any` on the raw iterator

Reply:
Done